### DISCUSSION TITLE
Incompatible route:list command with VSCode extension Laravel Artisan (Ryan Naddy)

### DISCUSSION THREAD

**Discussion by ajcastro** [laravel/framework#41090] [Ideas]
I am using Laravel Artisan extension by Ryan Naddy and I am using the `Artisan: Route List` command.
In Laravel 9 it is incompatible? Is there a way to use the old output of `php artisan route:list` command?

Laravel Artisan extension:
![image](https://user-images.githubusercontent.com/4918318/154611357-03078603-4df8-44c4-a34c-5772a3035b72.png)

Working in Laravel 8
![image](https://user-images.githubusercontent.com/4918318/154611381-10d286d0-78a0-4c74-b50a-234d09865e89.png)

Ugly in Laravel 9
![image](https://user-images.githubusercontent.com/4918318/154611396-c8ac6bd4-9b7f-44cf-95ba-75d50145c780.png)

**Comment by gguney** [comment]
If you write php artisan route:list both Laravel 8 and Laravel 9 you will see their outputs are different. In Laravel 8 it is table like output and probably extension is trying to parse the output like it is Laravel 8. So, extension should change its parser code or Laravel 9 should revert its output

Here you can see my outputs:
**Laravel 8**
![Screen Shot 2022-02-19 at 10 41 29](https://user-images.githubusercontent.com/16459896/154791857-32d1fff7-5f62-4b6f-9677-744cbc15769b.png)

**Laravel 9**
![Screen Shot 2022-02-19 at 10 41 07](https://user-images.githubusercontent.com/16459896/154791864-88615825-f30f-4b45-9479-a076e112589c.png)


If you look at the repository of the extension [Route List Command](https://github.com/TheColorRed/vscode-laravel-artisan/blob/master/src/commands/route/List.ts) you can see it has not updated for 4 years.

**Comment by TheColorRed** [comment]
The latest release (4/23/2023) has been updated to support this.
